src/index.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
-const _ = require('lodash');
 const core = require('./core');
 const RippleAPI = require('./api');
 
-module.exports = _.assign({}, core, {RippleAPI: RippleAPI});
+module.exports = {
+  RippleAPI,
+  _DEPRECATED: core   // WARNING: this will be removed soon
+};
```

Comment:
I'm actually a little confused as to how the tests all pass with this change. What's going on here?